packages/wallet/src/redux/sagas/messaging/__tests__/message-handler.test.ts

```diff
@@ -563,11 +595,11 @@ describe("message listener", () => {
         turnNum: 1
       };
 
-      expect(effects.put[0].payload.action).toMatchObject({
+      expect(effects.put[1].payload.action).toMatchObject({
         type: "WALLET.APPLICATION.OWN_STATE_RECEIVED",
         state: nextState
       });
-      expect(effects.put[1].payload.action).toMatchObject({
+      expect(effects.put[2].payload.action).toMatchObject({
```

Comment:
Also these kinds of diffs are annoying. @lalexgap any ideas on ways we could possibly handle these better? Maybe matching `put` calls based on the thing being put? e.g., "expect _some_ `put` to match object"

Reply:
We could create a helper that does that similar to `itSendsThisMessage`